x-pack/plugins/reporting/server/browsers/chromium/driver_factory/index.ts

```diff
@@ -104,6 +112,19 @@ export class HeadlessChromiumDriverFactory {
 
       const childProcess = {
         async kill() {
+          try {
+            const endMetrics = await devTools.send('Performance.getMetrics');
+            const { cpu, cpuInPercentage, memory, memoryInMegabytes } = getMetrics(
+              startMetrics,
+              endMetrics
+            );
+
+            apm.setCustomContext({ cpu, memory });
```

Comment:
I suspect you'll want to perform range queries, aggregate, etc on this so you'll want to use labels instead - [custom context](https://www.elastic.co/guide/en/apm/get-started/current/metadata.html#custom-fields) is not indexed while labels are

Reply:
@graphaelli But labels are equivalent to tags. Do you think it's logically correct to use labels here? It's something we can use aggregation for (like CPU cores), but here we are dealing with metrics.
The data we pass is fully random and distinct. Please correct me if I am wrong, but indexing this will increase the index size. Besides that, I can still filter transactions by the custom context fields.

Reply:
@dokmic we do want to perform aggregations on the data, which means it has to be indexed. The custom context is not indexed, and we can see the raw data, but in order to get charts of the data, it needs to be indexed. It will increase the index size very negligibly since it's numeric.